### PR TITLE
add favicon with github url, title and metadata for SEO rr/issue 53

### PR DESCRIPTION
**Pull request by rebecarubio**
Favicon folder added automatically inside the build folder and linked with the github link.
It shows in Chrome, but I checked and I can't see it in my IE and Firefox browsers.

 
not favicon ie
firefox_favicon

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-    <link
-        rel="stylesheet"
-        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
-        crossorigin="anonymous"
-    />
-    <!--
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
```
Comment (otech47) [inline public/index.html]: Please revert this link back to having each attribute on a seperate line. It is more readable that way

File: public/index.html [hunk 2]
```diff
@@ -25,12 +23,24 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <!-- Favicon images -->
+  <link rel="icon" href="https://github.com/setlife-network/website/files/3630402/favicon_io.zip">
```
Comment (otech47) [inline public/index.html]: You will need to unzip the zip file and find the .ico file inside

I think this guide may help explain as well: https://medium.com/@jenniferdobak/the-public-folder-and-favicons-in-create-react-app-8dc2cc1d492b

File: public/index.html [hunk 3]
```diff
@@ -25,12 +23,24 @@
...
+
+  <!-- title & metadata for SEO -->
+  <title>SetLife | People over Profit</title>
+  <meta name="title" property="og:title" content="SetLife | People over Profit" />
+  <meta name="image" property="og:image"
+    content="https://user-images.githubusercontent.com/4914611/65231213-81aa0700-dacf-11e9-9450-1fdad1f71b76.png" />
+  <meta name="description" property="og:description"
```
Comment (otech47) [inline public/index.html]: Made a last minute change to this copy if you could swap it out please:

“Learn valuable technical skills to upgrade your career or simply keep up with advancing technology”

Comment (rebecarubio) [reply public/index.html]: Ok, changed it.

**Pull request merged**